setup.py

```diff
@@ -39,26 +39,35 @@ def read_reqs(fname):
     req_path = os.path.join(here, fname)
     return [req.strip() for req in read(req_path).splitlines() if req.strip()]
 
-dev_reqs = read_reqs('requirements-dev.txt')
-extras_require = {"test": dev_reqs, "dev": dev_reqs}
 
-pip_too_old = False
-pip_message = ""
+s3_reqs = ["papermill[s3]"]
```

Comment:
A style comment. Perhaps flip to `reqs_s3` `reqs_gcs`. Should you ever have to parse by name it would be easier if all start `reqs_`.